weaviate/collection/classes/config.py

```diff
@@ -9,12 +9,40 @@
 
 
 class ConsistencyLevel(str, Enum):
+    """The consistency levels when writing to Weaviate with replication enabled.
+
+    Attributes:
+        ALL: Write to all replicas.
```

Comment:
Should this be `Require confirmation from all replicas`? 

My understanding is that the replication factor determines the `n` replicas, and the consistency level sets the min. confirmation level before the object becomes available / operation is confirmed.

Reply:
It's a subtle difference but I think yours is more accurate than mine so will change! `ALL` doesn't write to every node simultaneously but instead to a coordinator that distributes the writes, which ends up effectively writing to all nodes but not actually writing to all nodes